src/main/java/com/microsoft/aad/msal4j/Authority.java

```diff
@@ -36,6 +36,10 @@
     String authorizationEndpoint;
     String tokenEndpoint;
 
+    private final static String ADFS_AUTHORITY_FORMAT = "https://%s/%s/";
```

Comment:
ADFS authority format should be inside ADFS Authority, Authority class represents common authority